docs/analyzer/user_guide.md

```diff
@@ -1778,41 +1779,66 @@ optional arguments:
                         Show checkers only from the analyzers specified.
                         Currently supported analyzers are: clangsa, clang-
                         tidy.
-  --details             Show details about the checker, such as description,
-                        if available.
-  --profile {PROFILE/list}
-                        List checkers enabled by the selected profile.
-                        'list' is a special option showing details about
-                        profiles collectively.
-  --guideline GUIDELINE [GUIDELINE ...]
-                        List checkers that report on a specific guideline
-                        rule. Here you can add the guideline name or the ID of
-                        a rule. Without additional parameter, the available
-                        guidelines and their corresponding rules will be
-                        listed.
+  --details             Show details about the checker, such as status,
+                        checker name, analyzer name, severity, guidelines and
+                        description. Status shows if the checker is enabled
+                        besides the given labels. If the labels don't trigger
+                        a checker then the status is determined by the
+                        analyzer tool.
+  --label [LABEL]       Filter checkers that are attached the given label. If
```

Comment:
Please identify the format of the label (e.g.: `<label name>:<label value>`) and give a hint how the we can list the label values for a label (`CodeChecker checkers --label <label name>`), and in brackets add a command how we can list the available labels: `CodeChecker checkers --label`.

Reply:
I included label format, but I think a usage example here is not necessary. I think it has no additional value to give an example to every label about its own usage, if it is correctly described. And actually we have that at the hints at the bottom of the page. We have examples on other sub-commands as cross referencing, because their description is found in the help page of another command.